scene/gui/range.cpp

```diff
@@ -95,7 +95,14 @@ void Range::set_value(double p_val) {
 
 void Range::_set_value_no_signal(double p_val) {
 	if (!Math::is_finite(p_val)) {
-		return;
+		if (shared->allow_infinite_input || shared->allow_nan_input) {
+			if (Math::is_nan(p_val) && shared->allow_nan_input) {
+				shared->val = p_val;
+				return;
+			}
```

Comment:
If the value is infinite and allow nan is set but allow infinite is not set, this would allow it to be set, right?  Or is that expected behavior that allow nan allows anything, including infinite?